nixos/modules/programs/steam.nix

```diff
@@ -39,6 +65,20 @@ in {
       driSupport32Bit = true;
     };
 
+    security.wrappers = mkIf (cfg.gamescopeSession && gsCfg.capSysNice) {
+      # needed or steam fails
+      bwrap = {
+        owner = "root";
+        group = "root";
+        source = "${pkgs.bubblewrap}/bin/bwrap";
+        setuid = true;
+      };
+    };
+
+    programs.gamescope.enable = mkIf cfg.gamescopeSession true;
+    programs.gamescope.args = mkIf cfg.gamescopeSession [ "-e" ];
```

Comment:
Might be useful to add a comment explaining what this flag does.

Reply:
We should probably use the long-form flag `--steam` along with a comment.